portfolio/src/main/java/com/google/sps/servlets/ContextInitServlet.java

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.*;
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import javax.servlet.annotation.WebListener;
+import com.googlecode.objectify.ObjectifyService;
+import com.google.sps.data.Comment;
+
+@WebListener
+public class ContextInitServlet implements ServletContextListener {
+
+  @Override
+  public void contextInitialized(ServletContextEvent servletContextEvent) {
+    System.out.println("Initialize ObjectifyService...");
```

Comment:
TODO (for a future): use logging library (e.g. java.util.logging.Logger) instead of println.